tests/integration/helpers/hlr-lookup.spec.js

```diff
@@ -46,7 +46,7 @@ describe('Integration: HLR Lookup with callback', function () {
     return getTimeoutPromise(500)
     .then(function () {
       return Promise.all(promises);
-    });
+    })
```

Comment:
why you removed the comma here ?

Reply:
you mean semicolon, it was an eslint vialoation try to expand the code a bit to see.